charts/gardener/gardenlet/charts/runtime/templates/clusterrole-gardenlet.yaml

```diff
@@ -0,0 +1,293 @@
+{{- if .Values.global.gardenlet.enabled }}
```

Comment:
With https://github.com/gardener/gardener/pull/3535 the default resource lock was changed to leases. Currently I don't see any rule for leases. I think we should add a rule like:

```yaml
- apiGroups:
  - coordination.k8s.io
  resources:
  - leases
  verbs:
  - create
- apiGroups:
  - coordination.k8s.io
  resources:
  - leases
  resourceNames:
  - gardenlet-leader-election
  verbs:
  - get
  - watch
  - update
```